index.html

```diff
@@ -259,8 +259,53 @@ <h3>Deployment Scenarios</h3>
         demonstrators, were described with Thing Descriptions
         that are constrained to the <a>Core Profile</a>.
       </p>
+    </section>
 
+    <!-- Use Cases -->
+  <section id="profile-use-cases">
+    <h2>Use Cases and Requirements</h2>
+    <p>
+      A set of over 30 use cases for the Web of Things were contributed by stakeholders 
+      from multiple industries for various application domains.
+      These have been published in the WoT Use Cases and Requirements document [wot-use-cases]. 
+    </p><p>
+      Based on these use cases a set of requirements have been derived which drive the development 
+      of the W3C Web of Things specification family. 
+      Several of these domains require easy integration of devices from multiple vendors, 
+      in other words, out-of-the-box interoperability. 
+      However, the descriptive approach taken by the WoT specifications generally leads 
+      to a large variety of different protocols and data formats, 
+      which can work against out-of-the box interoperability. 
+    </p><p>
+      For example, a WoT Thing Description (TD) can in theory include a description based on a 
+      networking protocol unknown to a device that wishes to connect to it.  
+      To ensure interoperability without additional customization (e.g. by writing software or 
+      performing complex setup or configuration steps), 
+      the range of such choices needs to be limited to a finite set so that a consumer 
+      of a Thing Description can be sure it will be able to interact with any possible Thing. 
+      A finite set of customization choices is also important for implementing devices with a fixed code base. 
+      Defining such constraints leads to the profile mechanism and the WoT Core profile.
+      </p>
+      <p>
+      In addition to multiple vertical use cases that will use HTTP(S) for their implementations,
+      there are two key horizontal use cases that are addressed by this profile specification.
+      </p>
+      <section id="profile-use-case-MVSE">
+        <h2>Multi-Vendor System Integration - Out of the box interoperability</h2>
+          Users of devices want to process data from all devices that conform to a class of devices in a uniform way. 
+          They need a guarantee that he is able to correctly interact with all affordances of the Thing 
+          that complies with this class of devices. 
+          Different interpretations of the same Thing Description, that lead to different behaviour, should not be possible.
+          Users want to integrate devices in existing scenarios out of the box, i.e. with close to zero configuration tasks.
+      </section>
+        <p>
+          A major benefit of the WoT architecture and the Thing Description is the clear 
+          separation of the interaction model and the protocol binding.
+          This separation makes it possible to define common protocol agnostic 
+          rules that enable these use cases.
```

Comment:
This sentence feels out of place since the cross-protocol interworking use case was removed.

1. There is now only one use case listed
2. What kinds of protocol-agnostic rules does this separation make possible?

Reply:
I changed the text to:
This separation makes it possible to define common protocol agnostic 
          rules that enhance interoperability between different protocols.
          
Examples of rules/bindings are common brhavior and data structures for different protocols. 
Think of the sync and async actions, the same behavior could also be provided in other protocol bindings.